toolkit/partner-chains-cli/src/create_chain_spec/tests.rs

```diff
@@ -3,32 +3,31 @@ use super::PartnerChainRuntime;
 use crate::create_chain_spec::{CreateChainSpecCmd, INITIAL_PERMISSIONED_CANDIDATES_EXAMPLE};
 use crate::tests::runtime::{MockRuntime, TestSessionKeys};
 use crate::tests::{CHAIN_CONFIG_FILE_PATH, MockIO, MockIOContext};
-use crate::{CmdRun, ParsedPermissionedCandidatesKeys, verify_json};
+use crate::{CmdRun, KeyDefinition, verify_json};
 use colored::Colorize;
 use sidechain_slots::SlotsPerEpoch;
-use sp_core::{ed25519, sr25519};
-use sp_runtime::AccountId32;
 
 impl PartnerChainRuntime for MockRuntime {
-	fn create_chain_spec(config: &super::CreateChainSpecConfig) -> serde_json::Value {
+	type Keys = TestSessionKeys;
+
+	fn create_chain_spec(
+		config: &super::CreateChainSpecConfig<TestSessionKeys>,
+	) -> serde_json::Value {
 		serde_json::json!({
-			"session":config.pallet_partner_chains_session_config::<MockRuntime, _>(to_test_session_keys),
-			"sessionCommitteeManagement": config.pallet_session_validator_management_config::<MockRuntime, _>(to_committee_member),
+			"session":config.pallet_partner_chains_session_config::<MockRuntime>(),
+			"sessionCommitteeManagement": config.pallet_session_validator_management_config::<MockRuntime>(),
 			"sidechain": config.pallet_sidechain_config::<MockRuntime>(SlotsPerEpoch(13)),
 			"governedMap":config.governed_map_config::<MockRuntime>(),
 			"nativeTokenManagement":config.native_token_management_config::<MockRuntime>(),
 		})
 	}
-}
-
-fn to_committee_member(
-	keys: &ParsedPermissionedCandidatesKeys,
-) -> (AccountId32, (sr25519::Public, ed25519::Public)) {
-	(keys.account_id_32(), (keys.aura, keys.grandpa))
-}
 
-fn to_test_session_keys(keys: &ParsedPermissionedCandidatesKeys) -> (AccountId32, TestSessionKeys) {
-	(keys.account_id_32(), TestSessionKeys { aura: keys.aura.into(), grandpa: keys.grandpa.into() })
+	fn key_definitions() -> Vec<crate::KeyDefinition<'static>> {
+		vec![
+			KeyDefinition { name: "TestGrandpaLike", scheme: "ed25519", key_type: "ed25" },
+			KeyDefinition { name: "TestAuraLike", scheme: "sr25519", key_type: "sr25" },
+		]
+	}
```

Comment:
I keep using two keys, but these are not aura and gran. Similar story like in committee selection crates.